robottelo/host_helpers/cli_factory.py

```diff
@@ -27,7 +27,10 @@
 from robottelo import constants
 from robottelo import manifests
 from robottelo.cli.base import CLIReturnCodeError
+from robottelo.cli.proxy import CapsuleTunnelError
 from robottelo.config import settings
+from robottelo.host_helpers.system_tools import default_url_on_new_port
```

Comment:
@JacobCallahan Is this how we suppose to use the helpers or do u want them to be used via Host Class objects ?

Reply:
already resolved by the new approach